src/Track.cpp

```diff
@@ -372,7 +375,11 @@ bool Track::contains(Point* point) {
         ((point->getY() <= _start.getY()+1.0E-2 &&
           point->getY() >= _end.getY()-1.0E-2)
       || (point->getY() >= _start.getY()-1.0E-2 &&
-          point->getY() <= _end.getY()+1.0E-2)))) {
+          point->getY() <= _end.getY()+1.0E-2)) &&
```

Comment:
Oh man, what the heck is this?! We should really make this "1.0E-2" a macro or something. I know this is old code, but it should be cleaned up now that we are looking over it once again here.